easybuild/easyconfigs/n/NLopt/NLopt-2.6.1-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# 2019-06-05 John Dey jfdey@fredhutch.org fizwit@github.com - updated for CMake
+easyblock = 'CMakeMake'
+
+name = 'NLopt'
+version = '2.6.1'
+
+homepage = 'http://ab-initio.mit.edu/wiki/index.php/NLopt'
+description = """ NLopt is a free/open-source library for nonlinear optimization, 
+ providing a common interface for a number of different free optimization routines 
+ available online as well as original implementations of various other algorithms. """
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/stevengj/nlopt/archive']
+sources = [{
+    'download_filename': 'v%(version)s.tar.gz',
+    'filename': SOURCE_TAR_GZ
+}]
+checksums = ['66d63a505187fb6f98642703bd0ef006fedcae2f9a6d1efa4f362ea919a02650']
+
+builddependencies = [
+    ('CMake', '3.13.3'),
+    ('binutils', '2.31.1'),
+]
+
+configopts = ['-DBUILD_SHARED_LIBS=ON', '-DBUILD_SHARED_LIBS=OFF']
```

Comment:
Please reformat this with one set of configure options per line, to make it stand out this is an iterated build:

```python
configopts = [
    "-DBUILD_SHARED_LIBS=ON",
    "-DBUILD_SHARED_LIBS=OFF",
]